### ISSUE TITLE
merge ipad into desktop ui

### ISSUE THREAD

**Issue by yamsellem** [cheeaun/hackerweb#41]
This project is great ;-).

_Their is only one thing that bothers me_: the ipad ui is great and I don't get why it's not used for desktop. The 2 columns with a list pane on the left is perfect on desktop too.

Don't you think so?
Thanks.

**Comment by berbaquero** [comment]
@yamsellem: If you're on Mac, in the meantime, you could create a [Fluid](http://fluidapp.com/) app, and change the User Agent string into the iPad one and that would also work.

![screen shot 2015-06-24 at 13 47 57](https://cloud.githubusercontent.com/assets/1769951/8329174/a53c6310-1a77-11e5-9b49-b9342d10e59a.png)

I'm sure there's something similar for Windows and Linux.

**Comment by berbaquero** [comment]
Actually, @cheeaun, on a kinda related note: would you think creating an "Install" button on Firefox would be something you'd welcome for the app?

Using `window.navigator.mozApps.install('path/to/manifest')` using the manifest you already have for the Marketplace.

**Comment by cheeaun** [comment]
@berbaquero you mean showing the button on the app itself? Currently I'm leaning towards Chrome's implementation here: https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android . I haven't seen any updates from Mozilla/Firefox's side for a long time now.

**Comment by berbaquero** [comment]
Yeah, I mean showing an 'Install' button somewhere on the app, or in the homepage, similar to:

![screen shot 2015-06-24 at 21 47 30](https://cloud.githubusercontent.com/assets/1769951/8339851/ce1099bc-1aba-11e5-97f0-b0d9307d53c3.png)

> Reeddit's homepage, on Firefox only, of course.

**Comment by cheeaun** [comment]
@berbaquero ah, I see what you mean. As of now, not really thinking of implementing that. Is it working for you on Reeddit, as in users ask for it or using it?

**Comment by berbaquero** [comment]
@cheeaun actually, in Reeddit's case it was just a PR I accepted 😛

But I really like the idea of **explicitly** "installing directly from the browser".

Now that you mention it, I should probably put an event tracking on that button and see how it does, heh.
